src/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import cx from 'classnames';
+
+import RichText from 'RichText';
+
+type TimelineMilestoneProps = {
+  body?: string;
+  date: string;
+  linkHref?: string;
+  linkText?: string;
+  title: string;
+};
+
+type TimelineProps = {
+  className?: string;
+  milestones: TimelineMilestoneProps[];
+  title?: string;
+};
+
+export const Timeline = ({ className, milestones, title }: TimelineProps) => {
+  const classNames = cx('cc-timeline', {
+    [className]: className
+  });
+
+  return (
+    <div className={classNames}>
+      {title && <h3 className="cc-timeline__title">{title}</h3>}
+      <ol className="cc-timeline__list">
+        {milestones.map((item, index) => (
+          <li
+            // index used to ensure no two <dt><dd> combos have identical keys
+            // eslint-disable-next-line react/no-array-index-key
+            key={`${index}-${item.date}-${item.title}`}
```

Comment:
A workaround I've found for this eslint flag is to use `index + 1` and use a 1-based index - I'm just as happy with an eslint disable though to be honest, as it can be a frustrating rule!

Reply:
yeah it's annoying - it exists for good reason it's just a shame it can't read the context the index is in i.e. it'll be unique come what may